files/default/phpcheck

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ "$1" == "-h" ] ; then
```

Comment:
I recommend you use ``getopts`` and follow what we did in [logs-pull](https://github.com/osuosl-cookbooks/osl-backup/blob/master/files/default/logs-pull). I'd recommend we have an optional arg with ``-d`` which defaults to ``/var/www`` and ``-c`` which basically does what ``$1`` does.

Please do the same with the other script.